.github/workflows/image.yml

```diff
@@ -65,10 +65,7 @@ jobs:
             type=pep440,pattern={{raw}}
 
     - name: Free up disk space
-      uses: jlumbroso/free-disk-space@54081f138730dfa15788a46383842cd2f914a1be # v1.3.1
-      with:
-        tool-cache: true
-        docker-images: false
+      uses: openmerlin/dockerfile/.github/actions/clean-up
```

Comment:
this [action][1] is not active enough

[1]: https://github.com/jlumbroso/free-disk-space

Reply:
This action has been used by many opensource project, so if it's not a issue, pls don't change this.

such as:
https://github.com/apache/infrastructure-gha-allowlist-manager/blob/1d14c828fe21ce51adba1713664ec1d253671ad2/approved_patterns.yml#L161

Reply:
ok thanks, will revert this commit